onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2082,10 +2082,14 @@ Example 4:
       .Attr("stash_type",
             "type used for stash mean/inv_std_var",
             AttributeProto::INT, static_cast<int64_t>(ONNX_NAMESPACE::TensorProto_DataType_FLOAT))
+      .Attr("no_bias",
```

Comment:
There is no need for this flag since the input bias is optional. If bias is not passed, there is no bias.